dev-tools/roxctl.sh

```diff
@@ -28,7 +28,14 @@ fi
 
 cache=""
 if [[ -n "${KUBECONFIG}" ]]; then
-  cache="/tmp/$(md5 -q ${KUBECONFIG})"
+  if command -v md5sum &> /dev/null; then
```

Comment:
Looks like previously we only handled `md5`.  Do we want to prefer `md5` or `md5sum`?  My assumption is we would want to prefer `md5` and thus flip the if and the elif

Reply:
It doesn't make a difference. It's depends which OS you are using, on MacOS its `md5`, on fedora `md5sum`. It is only checked to avoid executing the OpenShift magic multiple times, thus reducing the roxctl execution times.